stdlib/public/core/Policy.swift

```diff
@@ -420,7 +420,7 @@ infix operator   & : MultiplicationPrecedence, BinaryInteger
 
 // "Additive"
 
-infix operator   + : AdditionPrecedence, AdditiveArithmetic, String, Strideable
+infix operator   + : AdditionPrecedence, AdditiveArithmetic, String, Array, Strideable
```

Comment:
Unrelated to PR: looks like GitHub syntax highlighting needs updating to handle the new operator syntax.